todo_app/data/trello_items.py

```diff
@@ -0,0 +1,84 @@
+# This code uses the 'requests' library to call trello API
+# http://docs.python-requests.org
+import requests
+import os
+
+list_ids = {"to_do":"6501688a993cb949a6478d07",
+            "doing":"6501688bc072b227f990762d",
+            "done":"6501688a1c964b99f5ea6d20"
+}
```

Comment:
It would be worthwhile making these environment variables like the Board id etc. - as it is I can select another board but it will still be trying to add items to your board!